dashboard/app/models/pd/workshop_constants.rb

```diff
@@ -89,5 +89,11 @@ module WorkshopConstants
     }.freeze
 
     CSF_201_PILOT_END_DATE = Date.new(2019, 5, 20)
+
+    COURSE_KEY_MAP = {
```

Comment:
is there anywhere more global we define these constants, or is this as far as we should hoist them?